lib/interface/cli/commands/hybrid/init.cmd.js

```diff
@@ -584,6 +592,38 @@ const initCmd = new Command({
             installationEvent: installationProgress.events.RUNTIME_INSTALLED,
         });
 
+        let appProxyIP;
+        installationPlan.addStep({
+            name: 'install app-proxy',
+            func: async () => {
+                appProxyIP = await installAppProxy({
```

Comment:
do `installationPlan.addContext('appProxyIp', appProxyIp);`
this will make sure it will be available after restart